src/types/settings.ts

```diff
@@ -31,6 +31,7 @@ export type TomSettings = {
 	preload					: boolean|string,
 	allowEmptyOption		: boolean,
 	closeAfterSelect		: boolean,
+	clearInputOnSelect		: boolean,
```

Comment:
To be consistent with the existing `closeAfterSelect`, should it be renamed to `clearAfterSelect`?
```suggestion
	clearAfterSelect		: boolean,
```